packages/create-astro/src/actions/typescript.ts

```diff
@@ -71,29 +74,76 @@ export async function typescript(
 	}
 }
 
-export async function setupTypeScript(value: string, { cwd }: { cwd: string }) {
-	const templateTSConfigPath = path.join(cwd, 'tsconfig.json');
-	try {
-		const data = await readFile(templateTSConfigPath, { encoding: 'utf-8' });
-		const templateTSConfig = JSON.parse(stripJsonComments(data));
-		if (templateTSConfig && typeof templateTSConfig === 'object') {
-			const result = Object.assign(templateTSConfig, {
-				extends: `astro/tsconfigs/${value}`,
-			});
+const FILES_TO_UPDATE = {
+	'package.json': async (
+		file: string,
+		options: { value: string; ctx: PickedTypeScriptContext }
+	) => {
+		// do not add astro check command to build script if option is not strictest
+		if (options.value !== 'strictest') return;
```

Comment:
I think we might want to apply this for `strict` as well? The user has said that they do want to use typescript at this point, so I think it's fair!

Reply:
Done! I also made some changes for test to reset fixtures back to original state more consistently.